Calculator.cpp

```diff
@@ -1,37 +1,38 @@
-#include<iostream>
+
+#include <bits/stdc++.h>
+
+#define int            long long
+#define float          double
+#define vi             vector<int>
+#define FIO            ios_base::sync_with_stdio(false); cin.tie(NULL);
+

```

Comment:
Ise ignore kr dena agr smj na aaye to...meri template me hai vrna useless hai bilkul

Reply:
ok